components/resources/wallet_strings.grdp

```diff
@@ -944,9 +944,12 @@
   <message name="IDS_BRAVE_WALLET_NFTS_TAB_EMPTY_STATE_DISCLAIMER" desc="NFT tab empty state disclaimer">Compatible with NFTs on Solana (SPL) and Ethereum (ERC-721).</message>
   <message name="IDS_BRAVE_WALLET_NFTS_TAB" desc="NFTs tab label">NFTs</message>
   <message name="IDS_BRAVE_WALLET_NFTS_TAB_HIDDEN" desc="Hidden label">Hidden</message>
+  <message name="IDS_BRAVE_WALLET_NFTS_TAB_SPAM" desc="Spam label">Spam</message>
   <message name="IDS_BRAVE_WALLET_NFTS_TAB_HIDE" desc="Hide menu item label">Hide</message>
   <message name="IDS_BRAVE_WALLET_NFTS_TAB_UNHIDE" desc="Unhide menu item label">Unhide</message>
   <message name="IDS_BRAVE_WALLET_NFTS_TAB_EDIT" desc="Hide menu item label">Edit</message>
+  <message name="IDS_BRAVE_WALLET_NFT_UNSPAM" desc="Unspam option label">Unspam</message>
```

Comment:
@mkarolin — I agree that `Unspam` isn't working here. But I don't think we'd need the article `a` in our replacement either, since spam is usually not used in the singular.

I'd suggest `Not spam` or, if space allows, `Remove from spam list`

Reply:
Thank you @rmcfadden3 🙏 either one sounds great!